interface.py

```diff
@@ -575,10 +584,11 @@ def get_track_download(self, track_id: str, quality_tier: QualityEnum) -> TrackD
             if content_length and int(content_length) < 1024:
                 raise self.exception(f"Track '{track_id}' appears to be corrupted (only {content_length} bytes available)")
             
-            # Check if content type is appropriate for audio
-            if content_type and not any(audio_type in content_type.lower() 
-                                       for audio_type in ['audio', 'octet-stream', 'mpeg', 'flac', 'application']):
-                raise self.exception(f"Track '{track_id}' does not contain valid audio content")
+            # Only check for obviously non-audio content types (like HTML error pages)
+            # Be more permissive with content type validation since audio can be served with various MIME types
+            if content_type and any(bad_type in content_type.lower() 
```

Comment:
No, I would rather check for the actual types that I want than checking types I don't expect.